models/borrowing.model.js

```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+  const Borrowing = sequelize.define("borrowings", {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    updatedBy: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    returned_date: {
+      type: DataTypes.DATE,
+      allowNull: true,
+    },
+  });
```

Comment:
Also, include a field to indicate if the book has been returned. This will be useful when you want to remind students that have not returned the books they borrow